app/src/main/java/net/gsantner/opoc/util/FileUtils.java

```diff
@@ -11,6 +11,8 @@
 package net.gsantner.opoc.util;
 
 
+import android.util.MutableInt;
```

Comment:
please don't use android classes in FileUtils - it should stay compatible with java-se